cpp/log/etcd_consistent_store-inl.h

```diff
@@ -625,14 +619,18 @@ void EtcdConsistentStore<Logged>::OnEtcdServingSTHUpdated(
 
 
 template <class Logged>
-util::Status EtcdConsistentStore<Logged>::RunOneCleanUpIteration(
-    const int64_t clean_up_to_sequence_number) {
-  CHECK_LE(0, clean_up_to_sequence_number);
+util::Status EtcdConsistentStore<Logged>::CleanupOldEntries() {
   if (!election_->IsMaster()) {
     return util::Status(util::error::PERMISSION_DENIED,
                         "Non-master node cannot run cleanups.");
   }
 
+  if (!serving_sth_) {
```

Comment:
In the original, I think this checking was done with the lock held in the ```CleanUpEntriesThread``` loop?